releases/net471/KnownIssues/514195-Targeting .NET Framework 4.7.1 will copy extra files to your bin directory.md

```diff
@@ -0,0 +1,24 @@
+# Targeting .NET Framework 4.7.1 will copy extra files to your bin directory
+
+## Symptoms
+
+When building an application that targets .NET Framework 4.7.1, the tooling will copy 12 assemblies into your bin directory if you consume one or more .NET Standard-based components.
+This will result in the bin output of your application to be larger and include more files.
```

Comment:
will result --> results
to be larger and include more files --> being larger and including more files